src/server/app.js

```diff
@@ -379,14 +472,32 @@ app.get('/officeholder/:officeholderId', (req, res, next) => {
     email: 'joe.berney@lanecounty-or.gov',
     meetings: 'Every Monday at 9am at Lance county Courthouse, Eugene, Oregon'
   }
+  
+  if(officeRes.rows != null && officeRes.rows[0] != null){
+	var sourceInfo = officeRes.rows[0];
+		officeholderVar.officeTitle = sourceInfo.officetitle,
+		officeholderVar.officeholderName = sourceInfo.name,
+		officeholderVar.termStart = String(sourceInfo.termstart),
+		officeholderVar.termEnd = String(sourceInfo.termend),
+		officeholderVar.nextElectionDate = String(sourceInfo.termend),
+		officeholderVar.phone = sourceInfo.contactphone,
+		officeholderVar.email = sourceInfo.contactemail,
+		officeholderVar.meetings = sourceInfo.contactmeeting
```

Comment:
Theres a syntax error here, I think you meant to use semicolons instead of commas. We actually don't have to use semicolons at all.